lib/Unexpected.js

```diff
@@ -898,15 +910,14 @@ Unexpected.prototype.throwAssertionNotFoundError = function (subject, testDescri
     });
 };
 
-Unexpected.prototype.lookupAssertionRule = function (subject, testDescriptionString, args) {
+Unexpected.prototype.lookupAssertionRule = function (subject, testDescriptionString, args, requireAssertionSuffix) {
     var that = this;
     if (typeof testDescriptionString !== 'string') {
         throw new Error('The expect function requires the second parameter to be a string.');
     }
-
     var handlers = this.assertions[testDescriptionString];
     if (!handlers) {
-        this.throwAssertionNotFoundError(subject, testDescriptionString, args);
+        return;
```

Comment:
Then it should also return null at the end (when handlers are found for the assertion string, but none have a macthing signature)?

But yeah, I also think it's a style thing :)